testing/tracker/APKBUILD

```diff
@@ -0,0 +1,68 @@
+# Contributor: Rasmus Thomsen <oss@cogitri.dev>
+# Maintainer: Rasmus Thomsen <oss@cogitri.dev>
+pkgname=tracker
+pkgver=2.2.1
+pkgrel=0
+pkgdesc="Personal search tool and storage system"
+url="https://wiki.gnome.org/Projects/Tracker"
+arch="all !armhf"
+license="GPL-2.0-or-later"
+makedepends="
+	bash-completion
+	dbus-dev
+	glib-dev
+	gobject-introspection-dev
+	icu-dev
+	json-glib-dev
+	libsoup-dev
+	libxml2-dev
+	meson
+	py3-setuptools
+	sqlite-dev
+	networkmanager-dev
+	vala"
+options="!check"
+subpackages="
+	$pkgname-dev
+	$pkgname-doc
+	$pkgname-lang
+	lib$pkgname:libs
+	$pkgname-bash-completion:bashcomp:noarch"
+source=https://download.gnome.org/sources/${pkgname}/${pkgver%.*}/${pkgname}-${pkgver}.tar.xz""
+builddir="$srcdir/$pkgname-$pkgver"
+
+build() {
+	cd "$builddir"
+	meson \
+		--prefix=/usr \
+		--sysconfdir=/etc \
+		--mandir=/usr/share/man \
+		--localstatedir=/var \
+		--buildtype=release \
+		-Dsystemd_user_services=no \
+		. output
+	ninja -C output
+}
+
+check() {
+	cd "$builddir"
+	ninja -C output test
+}
+
+package() {
+	cd "$builddir"
+	DESTDIR="$pkgdir" ninja -C output install
+}
+
+libs() {
```

Comment:
Because libs is already [defined][libs], this is redundant.

[libs]:https://git.alpinelinux.org/abuild/tree/abuild.in#n1746